.circleci/config.yml

```diff
@@ -66,3 +81,9 @@ workflows:
           context:
             - DockerHub
             - RubyGems
+      - update_jira:
+          context:
+            - DockerHub
+            - update-jira-webhook
+          requires:
+            - push_to_rubygems
```

Comment:
On some other places I have seen they add:
```
          filters:
            branches:
              only:
                - master
```

Should that be present here too?